README.md

```diff
@@ -208,51 +208,96 @@ The Filing API will run on `$(docker-machine ip):8080`
 The Public API will run on `$(docker-machine ip):8082`
 
 #### To run the entire platform
-Clone the [HMDA Platform UI](https://github.com/cfpb/hmda-platform-ui) repo and the [HMDA Platform Auth](https://github.com/cfpb/hmda-platform-auth) repo into sibling directories of this one. Your directory structure should look like this:
-```shell
-~/dev/hmda-project$ ls -la
-total 16
-drwxr-xr-x   6 lortone  staff   204B Jul 25 17:44 ./
-drwxr-xr-x   9 lortone  staff   306B Jul 25 17:50 ../
-drwxr-xr-x  22 lortone  staff   748B Jul 27 16:28 hmda-platform/
-drwxr-xr-x  25 lortone  staff   850B Jul 25 17:13 hmda-platform-ui/
-drwxr-xr-x  23 lortone  staff   796B Jul 28 17:15 hmda-platform-auth/
-```
 
-From the _`hmda-platform-ui`'s_ root directory, run `yarn`. (Get yarn [here](https://yarnpkg.com/lang/en/docs/install/) if you don't have it installed.)
+1. Clone [hmda-platform-ui](https://github.com/cfpb/hmda-platform-ui) and 
+    [hmda-platform-auth](https://github.com/cfpb/hmda-platform-auth) into the same
+    directory as hmda-platform.
 
-From _`hmda-platform`'s_ root directory, run the following:
+        ~/dev/hmda-project$ ls -l
+        drwxr-xr-x  22 lortone  staff   748B Jul 27 16:28 hmda-platform/
+        drwxr-xr-x  25 lortone  staff   850B Jul 25 17:13 hmda-platform-ui/
+        drwxr-xr-x  23 lortone  staff   796B Jul 28 17:15 hmda-platform-auth/
 
-```shell
-sbt clean assembly
-docker-compose up
-```
+1. Build hmda-platform-ui
 
-This will bring up all the HMDA Platform services. The first run may take several minutes.
+        cd hmda-platform-ui && \
+        yarn && \
+        cd ..
 
-Next, find your docker machine's endpoint.
+    **Note:** This requires [yarn](https://yarnpkg.com/lang/en/docs/install/) to be installed.
 
-```shell
-# Typically defaults to 192.168.99.100, which will be used in the following examples
-docker-machine ip dev
-```
+1. Build hmda-platform
+
+        cd hmda-platform && \
+        sbt clean assembly
+
+1. Launch the stack with Docker Compose
+
+        docker-compose up
```

Comment:
So this doesn't need the `--build` flag because you're assuming the reader is doing this for the first time? (I'm not clear on all of the docker commands)